packages/flutter/lib/src/widgets/icon.dart

```diff
@@ -21,6 +21,20 @@ import 'icon_theme_data.dart';
 /// Typically this is introduced automatically by the [WidgetsApp] or
 /// [MaterialApp].
 ///
+/// {@tool sample}
+///
+/// This example shows how to use [Icon] to create a simple icon in your flutter
+/// code.
+///
+/// ```dart
+/// Icon(
+///   Icons.add,
+///   color: Colors.white,
```

Comment:
White may not be the best choice. Most apps probably have a white background (it's the default after `flutter create`) and if somebody copy & pastes this example they will not see the icon. Maybe chose a color that's less likely to be the background?